### ISSUE TITLE
[ED-LOW ] Tooltip for filter "Accessibility statement" has slight typo

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-evaluation-tools-list#111]
I think the tooltip for the filter "Accessibility statement" is missing an apostrophe for "tools" or should be in the singular of "tool":

**Current:** `<img alt="Vendor provided information on tools accessibility" data-toggletip-content="Vendor provided information on tools accessibility"...>`

**Proposed Option 1:** `<img alt="Vendor provided information on tool's accessibility" data-toggletip-content="Vendor provided information on tool's accessibility"...>`

**Proposed Option 2:** `<img alt="Vendor provided information on tool accessibility" data-toggletip-content="Vendor provided information on tool accessibility"...>`
